analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/Sonar/ConditionEvaluatesToConstant.cs

```diff
@@ -37,27 +37,37 @@ namespace SonarAnalyzer.Rules.CSharp
 {
     internal sealed class ConditionEvaluatesToConstant : ISymbolicExecutionAnalyzer
     {
-        private static readonly ISet<SyntaxKind> omittedSyntaxKinds = new HashSet<SyntaxKind>
+        private const string MessageFormat = "{0}";
```

Comment:
Please revert all changes to this directory.
The class (or its base, or its partial form) will remain in the `SymbolicExecution` directory. This is intentional for now.
Codesmells will be fixed as `clean as we code` during individual rule migration.